x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/planner/ExpressionTranslators.java

```diff
@@ -41,28 +42,36 @@
 import org.elasticsearch.xpack.ql.querydsl.query.RegexQuery;
 import org.elasticsearch.xpack.ql.querydsl.query.ScriptQuery;
 import org.elasticsearch.xpack.ql.querydsl.query.TermQuery;
+import org.elasticsearch.xpack.ql.querydsl.query.TermsQuery;
 import org.elasticsearch.xpack.ql.querydsl.query.WildcardQuery;
 import org.elasticsearch.xpack.ql.tree.Source;
+import org.elasticsearch.xpack.ql.type.DataType;
+import org.elasticsearch.xpack.ql.type.DataTypes;
 import org.elasticsearch.xpack.ql.util.Check;
+import org.elasticsearch.xpack.ql.util.CollectionUtils;
 import org.elasticsearch.xpack.ql.util.Holder;
 
 import java.time.OffsetTime;
 import java.time.ZonedDateTime;
 import java.time.temporal.TemporalAccessor;
+import java.util.LinkedHashSet;
 import java.util.List;
+import java.util.Set;
+
 
 public final class ExpressionTranslators {
 
     public static final String DATE_FORMAT = "strict_date_time";
     public static final String TIME_FORMAT = "strict_hour_minute_second_millis";
 
-    
+
     public static final List<ExpressionTranslator<?>> QUERY_TRANSLATORS = List.of(
             new BinaryComparisons(),
             new Ranges(),
             new BinaryLogic(),
             new Nots(),
             new Likes(),
+            new InComparisons(),
```

Comment:
What's the behavior of wildcard string in `In` - are they supported or not? Should we check their existence and if we discover some, throw an error? Or better yet convert them into `Equals` and `And`.
I can see how somebody doing an equals against two patterns might be tempted to use an `In` with both of them instead.

Reply:
great question.
`In` is always exact match (+/- case-sensitivity for ES).
we use the wildcard function when matching multiple patterns: `wildcard(command_line, "*localgroup*", "*group*", "*user*")`.

there has been a little confusion by some users that have used `in` with wildcards, so we can change that behavior if that reduces friction. I've avoided that so far, because it would quietly change the behavior of existing syntax. But eventually it may make sense to switch.